projects/1.16.1/assets/woot/woot/lang/zh_cn.json

```
@@ -0,0 +1,535 @@
+  "info.woot.intern.0": "潜行右击以切换模式",
+  "info.woot.intern.1": "手动放置控制器和单元",
+  "info.woot.intern.cell": "单元",
+  "info.woot.intern.controller.count.0": "%2d个%s",
+  "info.woot.intern.controller.count.1": "1到%2d个%s",
+  "info.woot.intern.mode.build": "建造%s工厂",
+  "info.woot.intern.mode.form": "组建工厂",
+  "info.woot.intern.mode.validate": "检验%s工厂",
+  "info.woot.intern.other.count": "%2d个%s",
+  "info.woot.mobshard.0": "使用碎片攻击生物以捕获信息",
+  "info.woot.mobshard.1": "击杀生物以为碎片填充信息",
+  "info.woot.mobshard.2": "碎片必须在快捷栏中才能进行填充",
+  "info.woot.mobshard.3": "合成以清除信息",
+  "info.woot.mobshard.a.0": "[未编写]",
+  "info.woot.mobshard.a.1": "[编写完成]",
+  "info.woot.mobshard.b.0": "[击杀 %d/%d]",
+  "info.woot.mobshard.b.1": "[编写完成]",
+  "info.woot.output_tank": "输出流体槽: %s %d/%d mB",
+  "info.woot.output_tank.empty": "输出流体槽: 0/%d mB",
+  "info.woot.perk.crusher": "产生工业先锋精华液",
+  "info.woot.perk.crusher.0": "生产%d%%流体",
+  "info.woot.perk.efficiency": "消耗更少萌生原液",
+  "info.woot.perk.efficiency.0": "减少%d%%液体消耗",
+  "info.woot.perk.headless": "产生头颅",
+  "info.woot.perk.headless.0": "头颅掉落率：%d%%",
+  "info.woot.perk.looting": "更改抢夺等级",
+  "info.woot.perk.mass": "提升生物生成量",
+  "info.woot.perk.mass.0": "生成%d只生物",
+  "info.woot.perk.rate": "加快生成时间",
+  "info.woot.perk.rate.0": "减少%d%%生成时间",
+  "info.woot.perk.slaughter": "产生工业先锋粉红史莱姆液体和肉汤",
+  "info.woot.perk.slaughter.0": "生产%d%%流体",
+  "info.woot.perk.tier_shard": "产生用于升级工厂的精华",
+  "info.woot.perk.tier_shard.0": "%d次精华掉落机会",
+  "info.woot.perk.tier_shard.1": "- %s掉落几率%.02f%%",
+  "info.woot.perk.tier_shard.2": "精华权重：%d/%d/%d",
+  "info.woot.perk.xp": "生成经验",
+  "info.woot.perk.xp.0": "生成%d%%生物经验",
+  "info.woot.shard.0": "右击以获得经验",
+  "info.woot.shard.1": "潜行右击消耗整组",
+  "info.woot.squeezer.0": "内部：红：%dmB 黄：%dmB 蓝：%dmB 白：%dmB",
+  "info.woot.squeezer.glow": "对机器使用萤石粉以切换清空/精确模式",
+  "item.woot.black_dyecasing": "黑色染料框架",
+  "item.woot.black_dyeplate": "黑色染料板",
+  "item.woot.blue_dyecasing": "蓝色染料框架",
+  "item.woot.blue_dyeplate": "蓝色染料板",
+  "item.woot.brown_dyecasing": "棕色染料框架",
+  "item.woot.brown_dyeplate": "棕色染料板",
+  "item.woot.conatus_fluid_bucket": "萌生原液桶",
+  "item.woot.crusher_1": "压榨战斗 I 升级插件",
+  "item.woot.crusher_2": "压榨战斗 II 升级插件",
+  "item.woot.crusher_3": "压榨战斗 III 升级插件",
+  "item.woot.cyan_dyecasing": "青色染料框架",
+  "item.woot.cyan_dyeplate": "青色染料板",
+  "item.woot.debug": "调试工具（开发者物品）",
+  "item.woot.dye_die": "染料框架模具",
+  "item.woot.efficiency_1": "振奋锅炉 I 升级插件",
+  "item.woot.efficiency_2": "振奋锅炉 II 升级插件",
+  "item.woot.efficiency_3": "振奋锅炉 III 升级插件",
+  "item.woot.ench_plate_1": "新参者之板",
+  "item.woot.ench_plate_2": "达人之板",
+  "item.woot.ench_plate_3": "魔法师之板",
+  "item.woot.enchant_fluid_bucket": "液态魔咒桶",
+  "item.woot.gray_dyecasing": "灰色染料框架",
+  "item.woot.gray_dyeplate": "灰色染料板",
+  "item.woot.green_dyecasing": "绿色染料框架",
+  "item.woot.green_dyeplate": "绿色染料板",
+  "item.woot.guide.landing": "一本写给这大的，坏的，魔法多方块生物工厂的指南。就像烹饪锅一样，你会喜欢它或者讨厌它的--由你选择。",
+  "item.woot.guide.name": "Woot指南",
```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
  "item.woot.guide.name": "Woot 指南",
```